OPAL/si/src/main/scala/org/opalj/fpcf/AnalysisScenario.scala

```diff
@@ -295,774 +294,25 @@ class AnalysisScenario[A](val ps: PropertyStore) {
             // The match statement handles three main scheduling strategies: "SPS", "MPS", "IPMS", and "SPMS".
             // Each strategy defines how computations are scheduled into batches based on their dependencies and other constraints.
             scheduleStrategy match {
-                case "SPS" =>
+                case SinglePhaseScheduling.name =>
```

Comment:
Do not hard-code the strategies, instantiate the reflectively from the fully-qualified class name (see many implementations of this pattern around OPAL, e.g., in `InitialEntryPointsKey`)